package.json

```diff
@@ -6,7 +6,7 @@
     "s3",
     "stream"
   ],
-  "version": "0.7.0",
+  "version": "0.8.1",
```

Comment:
Please don't increment the version; I'll do that in a separate commit.

Reply:
Got it, rolled it back.